scripts/bootstrap-aio.yml

```diff
@@ -60,15 +82,12 @@
         src: "{{ rpco_cfg_src_path }}/{{ item.name }}"
         dest: "/etc/openstack_deploy/{{ item.name | regex_replace('.aio$', '') }}"
```

Comment:
not essential, but `` | regex_replace('.aio$', '')`` can come out now seeing as none of the files have that extension